go/batch-submitter/drivers/sequencer/encoding_test.go

```diff
@@ -265,15 +122,22 @@ func testAppendSequencerBatchParamsEncodeDecode(
 		TotalElementsToAppend: test.TotalElementsToAppend,
 		Contexts:              test.Contexts,
 		Txs:                   nil,
+		Type:                  sequencer.BatchTypeLegacy,
 	}
 
 	// Decode the batch from the test string.
 	rawBytes, err := hex.DecodeString(test.HexEncoding)
 	require.Nil(t, err)
 
 	var params sequencer.AppendSequencerBatchParams
+	params.Type = sequencer.BatchTypeLegacy
```

Comment:
this should be automatically inferred on decoding correct? i.e. it isn't read during decoding?